fuel-vm/src/storage/predicate.rs

```diff
@@ -40,6 +41,8 @@ use super::{
 /// operations. However, predicates, as defined in the protocol, cannot execute contract
 /// opcodes. This means its storage backend for predicate execution shouldn't provide any
 /// functionality.
+///
+/// TODO: blob storage should be implemented
```

Comment:
Should we do this? Should we do this now? Mentioned as an open issue in the PR body.

Reply:
TODO: link to an issue

Reply:
It is a separate feature and not part of this PR. If we don't have task for this feature, let's create it)

Reply:
https://github.com/FuelLabs/fuel-vm/issues/798